src/components/Header.js

```diff
@@ -1,10 +1,77 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import styled from 'styled-components';
 
-export default function Header() {
-	return <Wrapper>발신자 변경해주는 상단바</Wrapper>;
-}
+const Header = ({ userList, toggleUser, status }) => {
```

Comment:
받아온 값들을 요기서 받아오고 밑에 state선언이 필요없이 그냥 넣으면 더 코드가 짧아 질거 같아요!
```suggestion
const Header = ({ userName, userImg, toggleUser }) => {
```

Reply:
훨씬 짧아지고 조건문도 필요 없게 되겠네요!!